integrated_channels/integrated_channel/management/commands/remove_duplicate_learner_transmission_audit_records.py

```diff
@@ -0,0 +1,65 @@
+"""
+Transmits consenting enterprise learner data to the integrated channels.
+"""
+from logging import getLogger
+
+from django.apps import apps
+from django.contrib import auth
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.db.models import Max
+from django.utils.translation import gettext as _
+
+User = auth.get_user_model()
+LOGGER = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command which removes the duplicated transmission audit records for integration channels
+    """
+    help = _('''
+    Transmit Enterprise learner course completion data for the given EnterpriseCustomer.
+    ''')
+
+    def handle(self, *args, **options):
+        """
+        Remove the duplicated transmission audit records for integration channels.
+        """
+        # Find the latest record for each enterprise_course_enrollment_id that doesn't start with "course-v1:"
```

Comment:
Can we add the reasoning behind this in the help text of this management command